emData/generate_IR.py

```diff
@@ -128,7 +129,11 @@ def createDefinitions(wiresFiles='./LUTs/wires.dat'):
     file.close()
     os.remove(templateName)
 
-wiresFile = './LUTs/wires.dat'
+if len(sys.argv) < 2:
+    print("Usage: " + sys.argv[0] + " WIRES_FILE")
+    sys.exit(1)
+wiresFile = sys.argv[1]
+
```

Comment:
Could add an option here to specify where to write the output files.

Reply:
Updated all the scripts to use `argparse`. I think this is much cleaner, flexible, consistent, etc.